server/router.ts

```diff
@@ -195,6 +196,7 @@ function setupRouter(app, models, fetcher, viewCountCache: ViewCountCache) {
   router.post('/createInvite', passport.authenticate('jwt', { session: false }), createInvite.bind(this, models));
   // TODO: Change to GET /invites
   router.get('/getInvites', passport.authenticate('jwt', { session: false }), getInvites.bind(this, models));
+  router.get('/getOffences', getOffences.bind(this, models));
```

Comment:
This should use passport authenticate

Reply:
Yea, I was thinking the same, so I didn't added passport auth. 
I have updated to passport auth, if it doesn't require I'll remove it.

Reply:
I haven't used passport authenticate as the screen is public.